.github/workflows/coderabbit.yml

```
@@ -0,0 +1,18 @@
+
+on:
+  pull_request:
+    branches:
+      - main # Change this to your default branch
+
+jobs:
+  code-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)